sdkit/models/model_loader/latent_upscaler.py

```diff
@@ -0,0 +1,23 @@
+import torch
+from diffusers import StableDiffusionLatentUpscalePipeline
+
+from sdkit import Context
+from sdkit.utils import log
+
+
+
+def load_model(context: Context, **kwargs):
+    log.info('Load latent upscale model')
+    upscaler = StableDiffusionLatentUpscalePipeline.from_pretrained("stabilityai/sd-x2-latent-upscaler", torch_dtype=torch.float16)
+    upscaler.to(context._device)
+
+    if context._vram_usage_level != "high":
+        upscaler.enable_sequential_cpu_offload()   # GPU?
```

Comment:
I'm not sure CPU offload works if it's called after `.to(device)`. Will comment this out for now, and enable it back after some testing.